src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="no"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="apple-a1419"
+        class="card__picture"
+      >
+      <div class="card__names">
+        <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+        <div class="card__subtitle">Код товара: 195434</div>
+      </div>
+      <div class="card__feed">
+        <div class="stars stars--4">
```

Comment:
этот элемент находится внутри блока card, соответственно должен быть его элементом. При этом он может оставаться блоков для дочерних элементов.
```suggestion
        <div class="card__stars stars stars--4">
```